specs/multispec.yml

```diff
@@ -98,19 +102,39 @@ specs:
       cccp_job: "postgresql-{{ spec.short }}-centos7"
       latest_fedora: "f33"
 
+    "13":
+      version: "13"
+      prev_version: "12"
+      short: "13"
+      prev_short: "12"
+      common_image_name: "{{ spec.org }}/postgresql-{{ spec.short }}-{{ spec.prod }}"
+      rhel_image_name: "rhscl/postgresql-{{ spec.short }}-rhel7"
+      centos_image_name: "centos7/postgresql-{{ spec.short }}-centos7"
+      cccp_job: "postgresql-{{ spec.short }}-centos7"
+      latest_fedora: "f34"
+
+
 matrix:
   exclude:
     - distros:
         - fedora-31-x86_64
+        - fedora-34-x86_64
       version: "9.6"
     - distros:
         - fedora-31-x86_64
+        - fedora-34-x86_64
       version: "10"
     - distros:
         - rhel-7-x86_64
         - rhel-8-x86_64
         - centos-7-x86_64
+        - fedora-34-x86_64
       version: "11"
     - distros:
         - rhel-8-x86_64
+        - fedora-34-x86_64
       version: "12"
+    - distros:
+        - centos-7-x86_64
```

Comment:
Do we have the rpms in testing repositories yet? If so we could try building via:
https://github.com/sclorg/postgresql-container/blob/master/src/Dockerfile#L61

Reply:
I will ask @khardix .

Reply:
They should be available in the test repositories by now.